doltpy/core/dolt.py

```diff
@@ -86,11 +86,12 @@ class DoltCommit:
     Represents metadata about a commit, including a ref, timestamp, and author, to make it easier to sort and present
     to the user.
     """
-    def __init__(self, ref: str, ts: datetime, author: str, message: str):
+    def __init__(self, ref: str, ts: datetime, author: str, message: str, merge: Tuple[str, str] = None):
         self.hash = ref
         self.ts = ts
         self.author = author
         self.message = message
+        self.merge = merge
```

Comment:
what are we doing with this extra state?

Reply:
This just stores the tuple of parents so that the user can programmatically inspect if required, recalling our conversation with Turbine folks about the need to retrieve parents of merge commit.